joomla/joomla-cms
# [4.3] Media manager warning fixes

Pull Request for Issue # .

### Summary of Changes
- Add `.toString()` for the key and uid: https://github.com/joomla/joomla-cms/blob/94212433b370796c69cdd37c95402b9f40bb7b90/administrator/components/com_media/resources/scripts/components/app.vue#L6-L7
- Add the focused missing property: https://github.com/joomla/joomla-cms/blob/94212433b370796c69cdd37c95402b9f40bb7b90/administrator/components/com_media/resources/scripts/components/browser/items/item.es6.js#L200-L203
- Change the arguments of the `getContents`, `full` and `content` to boolean, default to false and fix all the calls:  https://github.com/joomla/joomla-cms/blob/94212433b370796c69cdd37c95402b9f40bb7b90/administrator/components/com_media/resources/scripts/app/Api.es6.js#L115


### Testing Instructions
- Run npm i
- run `npm watch:com_media`
- Open the admin site and go to media manager
- Open the browser console and refresh
- Observe the warnings
- Apply the PR
- run `npm watch:com_media`
- Open the admin site and go to media manager
- Open the browser console and refresh
- No warnings
- Check that uploading, editing, etc works as before


### Actual result BEFORE applying this Pull Request

![Screenshot 2023-03-04 at 14 05 04](https://user-images.githubusercontent.com/3889375/222903968-a2a67627-e438-4061-bb55-4cedefe83f09.png)

![Screenshot 2023-03-04 at 14 05 14](https://user-images.githubusercontent.com/3889375/222903978-4e0039df-494f-48d4-9f31-3a2246f3cb25.png)


### Expected result AFTER applying this Pull Request

No warnings 

### Link to documentations
Please select:
